src/index.ts

```diff
@@ -58,7 +58,7 @@ export { default as ProfileIcon } from './components/icons/Profile/Profile';
 
 // Hooks
 export { default as useAccordion } from './components/hooks/useAccordion/useAccordion';
-export { default as useForm } from './components/hooks/useForm/useForm';
+export * from './components/hooks/useForm';
```

Comment:
We need to check with @poteirard whether we're gonna hit any issues doing this 🧐

I recall when you tried @thegrinder the types could be imported nah?